lib/propcheck.ex

```diff
@@ -452,15 +449,13 @@ defmodule PropCheck do
                   end
                 end
 
-                reraise e, stacktrace
+                reraise e, __STACKTRACE__
```

Comment:
And then, an exception got reraised here. I even tried to remove the whole try-catch clause - the same result.
This makes me think that the issue is not in the PropCheck library or module, but most likely in the erlang's :proper.forall/2

Reply:
I wonder if this macro defined properly:
9e423393d8295b2faa6e1b384b71e/include/proper_internal.hrl#L49